.github/workflows/cxx-build.yml

```diff
@@ -160,22 +189,52 @@ jobs:
           key: deps-node-modules-${{ hashFiles('package-lock.json') }}
 
       - name: Install npm dependencies
-        run: npm ci
+        run: |
+          apt update
+          apt install -y cmake
+          npm install
 
       - name: Generate manage SDK source code
         run: |
           .github/workflows/utils.sh generate_cpp_manage_sdk_source_code
 
-      - name: Build CXX manage SDK 
+      - name: Build CXX Manage SDK
         run: |
+          apt install -y python3-pip python3.11-venv
+          python3 -m venv firebolt-env
+          source firebolt-env/bin/activate
+          pip install gcovr
+
           .github/workflows/utils.sh build_manage_cpp_sdk
 
-  dicovery_sdk:
+      - name: Test and Generate coverage report
+        run: .github/workflows/utils.sh generate_manage_sdk_coverage_report
+
+      - name: Display coverage report
+        uses: irongut/CodeCoverageSummary@v1.3.0
+        with:
+          filename: coverage/coverage.cobertura.xml
+          badge: true
+          fail_below_min: true
```

Comment:
See above RE: Not failing

Reply:
even if ctest fail. Git action steps won't be interrupted for now.